server/db/seeds.rb

```diff
@@ -9,12 +9,21 @@
 Restaurant.delete_all
 
 20.times do
-  Restaurant.create(
+  restaurant = Restaurant.create(
     name: Faker::Restaurant.name,
     cuisine: Faker::Restaurant.type,
     rating: Faker::Number.between(1, 3),
     accepts_10_bis: Faker::Boolean,
     address: Faker::Address.street_address,
     max_delivery_time: Faker::Number.between(30, 120)
   )
+
+  5.times do
+    Review.create(
```

Comment:
i believe you can use the factory here, the mock code should be the same
something like: `FactoryBot.create(Review, restaurant: restaurant)`

Reply:
Same goes for the restaurant

Reply:
While l don't really have an opinion on this, I stumbled upon this [article](https://thoughtbot.com/blog/factory_girl-for-seed-data) which might be interesting

Reply:
cool, and true.
If you choose to implement loading restaurants from an external service - this whole section won't be needed (as you wont need mock seed data but instead run a task to load them).